<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>158</width>
    <height>113</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alerta</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icones/cancelar.png</normaloff>../icones/cancelar.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(162, 162, 162);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:7pt &quot;Arial Rounded MT Bold&quot;;</string>
   </property>
   <property name="text">
    <string>PRODUTO N√ÉO CADASTRADO!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_Confirma">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-radius:10px;
background-color: #1E87F0;
color: #FFF

 }
QPushButton:hover{
border-radius:7px;
	background-color: rgb(85, 170, 255);

}</string>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
